### ISSUE TITLE
Non-paid Option

### ISSUE THREAD

**Issue by dadofsambonzuki** [lnbits/nostrnip5#21]
It would be good to have a non-paid option where admins can just manually add users. i.e. be disable the pay-to-add URL.

**Comment by jjim1982** [comment]
You can add users in the lnbits control panel

**Comment by dadofsambonzuki** [comment]
> You can add users in the lnbits control panel

Understood, generally - this is specific to the nostrnip5 ext though.

**Comment by jjim1982** [comment]
Yes, the extension management screen allows you to add users to domains that you set

**Comment by dadofsambonzuki** [comment]
Yep, I see I can do it manually. There is still no way to not add a fee when creating the domain. Surely it should be optional.

**Comment by tlindi** [comment]
yep - also after v1.0.0 "New address" seems gone complete bollocs
Press button an this is the view:
![Image](https://github.com/user-attachments/assets/b0521014-12e9-4bfb-84a1-ef5c3cd5addd)
